packages/gasket-plugin-git/lib/index.d.ts

```diff
@@ -0,0 +1,28 @@
+declare module '@gasket/cli' {
+  export interface CreateContext {
+    gitignore?: Gitignore;
+  }
+}
+
+export declare type GitignoreContent = {
```

Comment:
nit: I guess it's all lowercase file name, but this is so hard for me to to look at as single word without camelCasing Ignore:
```suggestion
export declare type GitIgnoreContent = {
```

non-blocking change request if you like the way you have it.